packages/atomic/src/components/facets/atomic-segmented-facet-scrollable/atomic-segmented-facet-scrollable.tsx

```diff
@@ -0,0 +1,56 @@
+import {Component, h} from '@stencil/core';
+import ArrowRightIcon from 'coveo-styleguide/resources/icons/svg/arrow-right-rounded.svg';
+import ArrowLeftIcon from 'coveo-styleguide/resources/icons/svg/arrow-left-rounded.svg';
+import {Button} from '../../common/button';
+
+@Component({
+  tag: 'atomic-segmented-facet-scrollable',
+  styleUrl: 'atomic-segmented-facet-scrollable.pcss',
+  shadow: true,
+})
+export class AtomicSegmentedFacetScrollable {
+  private horizontalScroll?: HTMLDivElement;
+
+  private horizonalSlider(arrowDirection: string) {
+    const container = this.horizontalScroll;
+    const pixelsToScroll = container ? container.clientWidth * 0.75 : 700;
+
+    if (container === null || !container) {
+      return;
+    }
+    if (arrowDirection === ArrowLeftIcon) {
+      container.scrollLeft -= pixelsToScroll;
+    } else {
+      container.scrollLeft += pixelsToScroll;
+    }
+  }
+
+  private renderArrow(arrowDirection: string) {
+    return (
+      <Button
+        style="square-neutral"
+        class="flex shrink-0 basis-8 justify-center items-center rounded"
+        ariaHidden="true"
+        onClick={() => this.horizonalSlider(arrowDirection)}
+      >
+        <atomic-icon class="w-3.5" icon={arrowDirection}></atomic-icon>
+      </Button>
+    );
+  }
+
+  render() {
+    return (
+      <div class="flex h-9">
+        {this.renderArrow(ArrowLeftIcon)}
+        <div
+          id="horizontalScroll"
+          class="wrapper-segmented flex flex-row overflow-x-scroll scroll-smooth"
```

Comment:
Since this div has styling, I would give it a part. I would do the same for the `div` above.

Reply:
the part's purpose is for implementers to be able to style our components. However, you're correct that you could reference the part instead of the id when looking up the element, so I think removing the `id` is a good idea.